app/views/content/terms_candidate.html.erb

```diff
@@ -2,87 +2,120 @@
 
     <p class="govuk-body"><%= govuk_link_to_with_utm_params('Chat online for help applying again', t('get_into_teaching.url_online_chat'), utm_campaign(params), current_candidate&.current_application.present? ? current_application.phase : t('layout.support_links.not_logged_in')) %>.</p>
 
-    <h3 class="govuk-heading-m">Making changes to your application</h3>
+    <h3 class="govuk-heading-s">Making changes to your application</h3>
 
-    <p class="govuk-body">Once you submit your application it’s sent to your teacher training providers straight away.</p>
+    <p class="govuk-body">Once you submit your application it’s sent to the teacher training provider you applied to.</p>
 
-    <p class="govuk-body">Contact us within 5 working days at <%= bat_contact_mail_to %> if you need to
-      change something.</p>
+    <p class="govuk-body">Contact us within 5 working days at <%= bat_contact_mail_to %> if you need to change something on your application once you’ve applied for a course.</p>
 
     <p class="govuk-body">You can ask us to update your personal or contact details any time.</p>
 
-    <h3 class="govuk-heading-m">Withdrawing your application</h3>
+    <h3 class="govuk-heading-s">Withdrawing your application</h3>
 
     <p class="govuk-body">You can withdraw your application at any point, even after you’ve accepted an offer.</p>
 
     <p class="govuk-body">
       <%= govuk_link_to('Sign in to your account to withdraw an application', candidate_interface_sign_in_path) %>.</p>
 
-    <h3 class="govuk-heading-m">Application outcome</h3>
+    <p class="govuk-body">You must not withdraw more than 20 applications in a 12 month period starting from October every year. For example, between October 2023 and October 2024, you must not withdraw more than 20 applications.</p>
+
+    <h3 class="govuk-heading-s">Application outcome</h3>
+
+    <p class="govuk-body">If your teacher training provider does not make a decision about your application by their deadline, your application to their course cannot progress. Providers normally have 40 working days to make a decision, but at certain times of the year this may change.</p>
+
+    <p class="govuk-body">Once you receive decisions from all of the training providers you applied to, you have 10 working days to accept or decline any offers. If you do not respond to your offers within this timeframe, they’ll be automatically declined.</p>
+
+    <p class="govuk-body">If a training provider wants to make a change to an offer you have accepted, you will be contacted to get your permission.</p>
+
+    <h3 class="govuk-heading-s">Criminal convictions</h3>
+
+    <p class="govuk-body">You’ll be asked about any criminal convictions in the application form. You'll need to have an enhanced DBS check if you accept a place on a course. This will show up spent or unspent criminal convictions. </p>
```

Comment:
🕵️ 
```suggestion
    <p class="govuk-body">You’ll be asked about any criminal convictions in the application form. You’ll need to have an enhanced DBS check if you accept a place on a course. This will show up spent or unspent criminal convictions. </p>
```